wbooks/src/app/screens/dashboard/components/Book/index.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import './style.css';
+
+class Book extends Component{
+  render(){
+    return(
+      <div className="book">
+        <img className="book-img" alt={this.props.title} src={this.props.image_url}/>
+        <h2 className="book-title">{this.props.title}</h2>
+        <p className="book-author">{this.props.author}</p>
+      </div>
+      );
+  }
+
+}
+
+export default Book;
```

Comment:
Define `propTypes` for this class
check this: https://reactjs.org/docs/typechecking-with-proptypes.html

also, if you only need the render method, you should declare Book as a const

```
const Book = (props) => (
    <div...
)
```